packages/astro/tsconfig.json

```diff
@@ -9,7 +9,7 @@
     "forceConsistentCasingInFileNames": true,
     "inlineSources": false,
     "isolatedModules": true,
-    "moduleResolution": "node",
+    "moduleResolution": "Bundler",
```

Comment:
We need to update this to properly resolve imports from packages that use export maps (like `astro/config)